src/serve/index.ts

```diff
@@ -0,0 +1,48 @@
+import { EmulatorServer } from "../emulator/emulatorServer";
+
+const _ = require("lodash");
+
+const logger = require("../logger");
+
+const TARGETS: {
+  [key: string]:
+    | EmulatorServer
+    | { start: (o: any) => void; stop: (o: any) => void; connect: () => void };
+} = {
+  hosting: require("./hosting"),
+  functions: require("./functions"),
+  database: require("./database"),
+  firestore: require("./firestore"),
+};
+
+const _serve = async function(options: any) {
+  const targetNames = options.targets;
+  options.port = parseInt(options.port, 10);
+  await Promise.all(
+    _.map(targetNames, function(targetName: string) {
```

Comment:
Prefer arrow functions in this case (`_.map(targetNames, (targetName: string) => {`)

Same below.